contracts/dev-utils/contracts/src/TransactionDecoder.sol

```diff
@@ -0,0 +1,32 @@
+/*
+
+  Copyright 2019 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.5.5;
+pragma experimental ABIEncoderV2;
+
+import "./LibTransactionDecoder.sol";
+
+
+// solhint-disable no-empty-blocks
+contract TransactionDecoder is
```

Comment:
Same question as above. What does this pattern do for us?

Reply:
Same logic as https://github.com/0xProject/0x-monorepo/pull/2161#discussion_r325899210